src/components/DataAccessLayer.ts

```diff
@@ -16,24 +19,7 @@ export function getQuestionsBasedOnEnv() {
   return getAllQuestions();
 }
 
-export const getAllQuestionsFromApi = async () => {
+export const getAllQuestionsFromApi = async (): Promise<MCQuestion[]> => {
   const allQuizzes = await NetworkCalls.getQuiz();
```

Comment:
wonder if still need networkcalls...?

Reply:
I didn't remove them since the implementation of the data being sent from the parent component hasn't been implemented yet. This was helpful in determining if the app breaks if the actual data was passed in, and/or if the tags conversion was working as expected.